projects/ng-tailwind/src/components/ngt-modal/ngt-modal.component.ts

```diff
@@ -48,17 +46,19 @@ export class NgtModalComponent {
         });
     }
 
+    public ngAfterViewInit(): void {
+        NgtModalHeaderComponent.closeModalByHeader.subscribe(() => this.close());
```

Comment:
(Atenção) Essa forma de subscribe não é interessante. Se não for implementada com o unsubscribe correto, pode aumentar o consumo de memória pelo navegador, além de gerar problemas de lógica no próprio componente.